### DISCUSSION TITLE
Creación de cuenta de azure

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#503] [Q&A] [answered]
Buenas tardes, disculpen para crear la cuenta de azure hay que colocar tarjeta de crédito¿? Habrá una manera que se pueda sin necesidad de colocar la tarjeta, de lo contrario, ¿Acepta tarjeta de Banamex?

**Comment by SandroBorrallez** [comment]
Hola Gloria, me paso lo mismo, que me pide datos de tarjeta, lo que hice fue cerrar la pagina de Azure cuando me pidió los datos de la tarjeta, y volví a abrir la página iniciando sesión con la cuenta que ya se registra, y posteriormente me fui a la opción Accede a las ventajas para alumno y sigues los pasos y listo!.

Espero te sea de ayuda.

**Reply by GloriaCoralCerecedo** [reply]
Gracias!

**Comment by ismagician** [comment, marked as answer]
Para la cuenta student no es necesario una tarjeta de crédito, solo te pide un número telefónico y puedes usar la cuenta de LaunchX para registrarte (anteriormente yo me registré con una cuenta de Innovaccion Virtual, supongo que aun sigue funcionando)
https://azure.microsoft.com/es-es/free/students/
Te paso el link del tutorial de Innovaccion Virtual para la creación de la cuenta  de Azure
https://www.youtube.com/watch?v=7AktCi6YM10

**Reply by GloriaCoralCerecedo** [reply]
Gracias

**Reply by GloriaCoralCerecedo** [reply]
Me funciono muy bien muchas gracias!

**Reply by ismagician** [reply]
De nada :v:

**Reply by lfsandovalruiz** [reply]
Muchas gracias amigo me ayudó bastante aquí tiene su like 👍
